src/nvim/misc1.c

```diff
@@ -2699,6 +2699,7 @@ void preserve_exit(void)
   }
 
   ml_close_all(false);              // close all memfiles, without deleting
+  stream_set_blocking(input_global_fd(), true);  //normalize stream (#2598)
```

Comment:
`getout()` further down calls `mch_exit()`, so stream normalizing will be repeated there. In case this function is called recursively, `mch_exit()` will not be reached, because we exit at line 2682. Is it still a problem to move this call before the `exit()` call to normalize only once?